src/Twig/CookieTwigExtension.php

```diff
@@ -3,34 +3,42 @@
 namespace Kunstmaan\CookieBundle\Twig;
 
 use Doctrine\ORM\EntityManagerInterface;
+use Kunstmaan\AdminBundle\Helper\DomainConfiguration;
+use Kunstmaan\CookieBundle\Entity\Cookie;
 use Kunstmaan\CookieBundle\Entity\CookieConfig;
 use Kunstmaan\CookieBundle\Entity\CookieType;
 use Kunstmaan\CookieBundle\Helper\LegalCookieHelper;
 use Symfony\Component\HttpFoundation\Request;
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFunction;
 
 /**
  * Class CookieTwigExtension
  *
  * @package Kunstmaan\CookieBundle\Twig
  */
-class CookieTwigExtension extends \Twig_Extension
+class CookieTwigExtension extends AbstractExtension
 {
     /** @var EntityManagerInterface */
     private $em;
 
     /** @var LegalCookieHelper */
     private $cookieHelper;
 
+    /** @var DomainConfiguration */
+    private $domainConfiguration;
+
     /**
      * CookieTwigExtension constructor.
      *
      * @param EntityManagerInterface $em
      * @param LegalCookieHelper      $cookieHelper
      */
-    public function __construct(EntityManagerInterface $em, LegalCookieHelper $cookieHelper)
+    public function __construct(EntityManagerInterface $em, LegalCookieHelper $cookieHelper, DomainConfiguration $domainConfiguration)
```

Comment:
Should it be better to typehint against the `Kunstmaan\AdminBundle\Helper\DomainConfigurationInterface` interface here?